src/results/DownloadContainer.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import idx from 'idx';
+import { RootState } from '../state/state-types';
+import DownloadView from './DownloadView';
+import { Column } from '../model/types/ColumnTypes';
+import { FileFormat, fileFormatToContentType } from './types/resultsTypes';
+import { getDownloadUrl } from '../utils/apiUrls';
+import fetchData from '../utils/fetchData';
+
+export const replaceExcelWithTsv = (fileFormat: FileFormat) =>
```

Comment:
This could be made more generic by dissociating it from the name of data types, renaming it to `getPreviewFileFormat`. This means that if file format changes later and there's another file format which is not supported by preview we can just add it to that method.

Reply:
Makes sense. Nice future proofing.